armi/tests/test_runLog.py

```diff
@@ -43,6 +51,47 @@ def test_invalidSetVerbosityByString(self):
         with self.assertRaises(KeyError):
             runLog.setVerbosity("taco")
 
+    @unittest.skipIf(
+        context.MPI_COMM is None, "mpi4py or MPI libraries are not installed."
+    )
+    def test_caplogBasicParentRunLog(self):
+        """A basic test of the logging of the parent runLog"""
+        with self._caplog.at_level(logging.INFO):
+            runLog.setVerbosity("info")
+            runLog.debug("This is invisible.")
+            runLog.info("one")
+            runLog.important("two two\n")
+            runLog.warning("three three three")
+            runLog.error("four four four four")
+
+        messages = [r.message for r in self._caplog.records]
+        assert len(messages) > 0
+        assert "one" in messages[0]
+        assert "two two" in messages[1]
+        assert "three three three" in messages[2]
+        assert "four four four four" in messages[3]
```

Comment:
Ack! Sorry, yes, `self.assertX`.  I keep forgetting about those. I just finished a big push to move away from those at my last job and keep trying to remember we are using them here. I'll fix those.

(I will take a look at ditching the two log classes in a second. I thought to preserve them because I can't see if they are being used in the plugins. Give me a minute on an opinion there.)

Reply:
you're moving away from the unittest assert methods? Why? Do tell!

Reply:
Oh, sorry, there was no exciting reason. There were some tools in pytest we wanted to use that didn't play well with `unittest`, so we dropped all usages of `unittest`. That's all.  

Pytest works fine with plain `assert`s, but the `self.assertX` comes from the `unittest.TestCase` class.